### ISSUE TITLE
Warnings when running qwen3-next-80B

### ISSUE THREAD

**Issue by Athanaze** [Mega4alik/ollm#9]
```
qwen3-next-80B DiskCache is not supported at the moment. Using default DynamicCache instead
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
```

**Comment by Mega4alik** [comment]
Thanks for the question. I noticed some buggy behavior with MoE models when using DiskCache, and rolled back. It uses default KVCache of transformers library for now. I want to learn more about your case, how large is your context size and what are your PC characteristics? 

On the masking side, it should not affect the inference. 

Thanks!
